Automattic/vip-go-mu-plugins
# Update Query Monitor 3.6.5 => 3.7.1

## Description

This updates the Query Monitor plugin from 3.6.5 to 3.7.1.

## Changelog Description

### Query Monitor updated to version 3.7.1.

Full changelog here: https://github.com/johnbillion/query-monitor/releases

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

**Important:** This introduces [a new feature which exposes the SQL query logs to suitably authenticated users during REST API requests](https://querymonitor.com/blog/2021/05/debugging-wordpress-rest-api-requests/). There is no change to the privileges required to see QM's data, but this will need testing on VIP before deploying because AFAICT [VIP uses a non-standard structure for the database queries logged in the `wpdb` class](https://github.com/automattic/vip-go-mu-plugins/blob/9180eab0ff8a2792325e8a5decf59e4a25c28e56/vip-helpers/vip-query-monitor.php).
